### DISCUSSION TITLE
Can't open Obsidian workspace in Zettlr

### DISCUSSION THREAD

**Discussion by AngeloHulshout** [Zettlr/Zettlr#5334] [Q&A]
I have a fresh installation of Zettlr, and tried to open a workspace by selecting an Obsidian folder, filled with .md files.
This did not work, and the logging shows:

Could not open root /Users/angelo/Documents/Projecten/ObsidianVault/Angelo because it was an ignored directory/file

What is wrong here? The criteria for ignoreDirs does not apply as far as I can tell.

**Comment by AngeloHulshout** [comment]
Looks like I found it. This happens with workspaces that have been put 'offline' by iCloud drive on my MacBook. After downloading those ('online') it works fine. Could it be that Zettler considers these directories ignored instead of inaccessible when downloading takes too long?

**Reply by nathanlesage** [reply]
It sounds as if the folder didn't actually physically exist before. Sounds like it just couldn't find the folder? Feel free to observe and add new info if necessary, but it's good to hear that it now works.

**Reply by AngeloHulshout** [reply]
Folders and files that are put 'offline' by iCloud Drive are invisible to Zettlr (and some other applications)/ That's why it thinks the folder does not exist. I don't know if there is a way around this, I solved it by moving the workspaces to a location  not managed by iCloud drive.

**Comment by hittjwX** [comment]
I'm finding Zettlr great for editing long papers inside the Obsidian vault. However, I only open subfolders in the vault. This works much better and gives me control over a folder with materials for one paper rather than all the notes in my vault.
